src/Voron/Impl/Paging/AbstractPager.cs

```diff
@@ -37,6 +37,13 @@ public abstract unsafe class AbstractPager : IDisposable, ILowMemoryHandler
 
         public Action<PagerState> PagerStateChanged;
 
+        public Func<long> AllocatedInBytesFunc;
+
+        public long GetAllocatedInBytes()
+        {
+            return AllocatedInBytesFunc?.Invoke() ?? 0;
```

Comment:
Why? We can get _this_ information from the pager state.

Reply:
The pager state contains the file size

Reply:
So why return 0?

Reply:
done